e2e/examples.spec.ts

```diff
@@ -1,87 +1,45 @@
 import { test, expect } from '@playwright/test';
-test.beforeEach(async ({ page }) => {
-  const { SPLUNK_USER: user, SPLUNK_PASSWORD: password, CREATE_ACCOUNT: createAccount } = process.env;
+import { goToDataSet } from './utils';
 
-  // Login
+test.beforeEach(async ({ page }) => {
   await page.goto('/');
-  await page.getByPlaceholder('Username').fill(user || 'AAAA');
-  await page.getByPlaceholder('Password',  { exact: true }).fill(password || 'BBBB');
-  await page.getByRole('button', { name: 'Sign In' }).click();
-
-  await page.getByLabel('Security Data Lake Add-On for Splunk').click()
-
-  console.log("Create account: ", createAccount, ", set to true to create account")
-  if ( createAccount == 'true') {
-    // Go to configuration
-    await page.goto('/en-GB/app/TA_dataset/configuration');
-
-    // Check that we are on the configuration page
-    await expect(page).toHaveTitle(/Configuration/);
-
-    // Open dialog
-    const locAddDialog = page.locator('#accountTab').getByRole('button', { name: 'Add' });
-    await locAddDialog.click();
-
-    // Setup locators
-    const locAccount = page.locator('div').filter({ hasText: /^Account nameEnter a unique name for this account\.$/ }).locator('[data-test="textbox"]');
-    const locUrl = page.locator('div').filter({ hasText: /^URLEnter DataSet URL\.$/ }).locator('[data-test="textbox"]');
-    const locReadKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Read Access KeyRequired to enable inputs and SPL comand. Include tra' }).locator('[data-test="textbox"]');
-    const locWriteKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Write Access KeyRequired to enable alert action. Include trailing hy' }).locator('[data-test="textbox"]');
-
-    // Read env with values
-    const { DATASET_URL: datasetUrl, DATASET_LOG_ACCESS_READ: datasetReadKey, DATASET_LOG_ACCESS_WRITE: datasetWriteKey } = process.env;
-
-    // Fill in values
-    const accountName = 'QaTr' + (Math.random() * 1e18)
-    await locAccount.fill(accountName);
-    await locUrl.fill(datasetUrl || '');
-    await locReadKey.fill(datasetReadKey || '');
-    await locWriteKey.fill(datasetWriteKey || '');
 
-    // Click on Add
-    const locAddConfirm = page.locator('[data-test="footer"]').getByRole('button', { name: 'Add' })
-    await locAddConfirm.click();
-
-    // await page.getByLabel(accountName).click()
-  }
+  await goToDataSet(page);
 });
 
+
 test('Check example page', async ({ page }) => {
-  await page.goto('en-GB/app/TA_dataset/dataset_by_example');
+  console.log("Go to example page");
```

Comment:
It's uncommon to see such informational logs in tests. If the steps from playwright are not clear, it might be better to focus on that and wrap the step in a function.

Reply:
I have feeling that I can see what is going on.

From my perspective it's better to have:
```js
console.log("Check that input field is filled with ", expectedValue);
```
rather then just comment:
```js
// Check that the input field is filled correctly
```

Reply:
If an assertion fails, Playwright will give an error like "This element has value XXX, but we expected YYY." so for me, all such `console.log`s seem redundant. But sure, keep them around if you find them useful.